hooks-admin.php

```diff
@@ -54,6 +54,10 @@
 add_action( 'admin_menu', '\Pressbooks\Admin\Diagnostics\add_menu', 30 );
 add_action( 'init', [ '\Pressbooks\Admin\SiteMap', 'init' ] );
 add_action( 'init', '\Pressbooks\Admin\Laf\remove_emoji' );
+
+// Custom Font Actions
+add_action( 'admin_post_pb_save_custom_fonts', '\Pressbooks\Admin\CustomFonts\handle_form_submission' );
```

Comment:
Add form submission handler action. I think this is the right place for that, but not 100% sure.